src/hooks/screen/useChatScreenHook.ts

```diff
@@ -229,7 +259,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   React.useEffect(() => {
     initChatData();
-  }, [localDb, chat, selectedChannel]);
+  }, [localDb, chat, otherListener[`chat_${selectedChannel?.id}`], selectedChannel]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependencies of the `useEffect` hook have been updated to include `otherListener[`chat_${selectedChannel?.id}`]`. This means the effect will re-run whenever the selected channel changes or the corresponding chat listener changes. This seems reasonable if you want to initialize chat data whenever these values change. However, make sure that this doesn't lead to unnecessary re-renders or side effects.


<!-- This is an auto-generated comment by OSS CodeRabbit -->